addons/account/models/account_move.py

```diff
@@ -3964,6 +3976,17 @@ def action_open_business_doc(self):
             'target': 'current',
         }
 
+    def action_update_taxes(self):
+        for line in self.line_ids.filtered(lambda l:l.display_type not in ('line_section', 'line_note', 'payment_term')):
+            if line.exists():
+                line._compute_tax_ids()
+                line._compute_account_id()
+        if self.partner_id:
+            self.message_post(body=_("Product taxes have been recomputed according to fiscal position %s.",
+                self.fiscal_position_id._get_html_link() if self.fiscal_position_id else "")
+            )
```

Comment:
Not sure this is needed since it's an action done by the user and every change of importance should already be logged in the chatter per field basis